src/assets/assets.controller.ts

```diff
@@ -142,4 +150,30 @@ export class AssetsController {
 
     res.json({ assets });
   }
+
+  @ApiExcludeEndpoint()
+  @Post('update-verified')
+  @UseGuards(ApiKeyGuard)
+  async update_verified(
+    @Body(
+      new ValidationPipe({
+        errorHttpStatusCode: HttpStatus.UNPROCESSABLE_ENTITY,
+        transform: true,
+      }),
+    )
+    dto: UpdateVerifiedAssetsDto,
+    @Res() res: Response,
+  ): Promise<void> {
+    const missing: VerifiedAssetMetadataDto[] = [];
+    for (const asset of dto.assets) {
+      const result = await this.assetsService.updateVerified(asset);
```

Comment:
Should `updateVerified` and `deleteVerified` be in a transaction together? That way if anything breaks, we get a clean atomic rollback

Reply:
yea thought about this. my only concern was that the transaction would be too big and interrupt other API stuff with a big SQL job. I think if a partial update went through it wouldn't be the worst thing in the world. Its idempotent after all so it could just be run again. Wdyt?